workbench/app/Console/StatCommand.php

```
@@ -25,140 +25,84 @@ class StatCommand extends Command
     /**
      * Execute the console command.
      */
-    public function handle(Province $province): void
+    public function handle(Province $province): int
```

PHPMD (2.15.0)

Avoid unused parameters such as '$province'. (Unused Code Rules)

(UnusedFormalParameter)